### PR TITLE
fix: add note preventing user from changing nameserver

### LINKED ISSUE #11080: Update "Configuring nameservers using the administrative shell" to include a note about 127.0.0.1
### Issue overview

Sometimes customers want to change their nameservers. They simply read [Configuring DNS nameservers](https://help.github.com/en/enterprise/2.19/admin/installation/configuring-dns-nameservers#configuring-nameservers-using-the-administrative-shell) and start making changes to the */etc/resolvconf/resolv.conf.d/head* file. We have seen some cases where customers just remove this:

```
nameserver 127.0.0.1
```

And got a ton of `500` errors after restarting `resolvconf`.

This is warned in the */etc/resolvconf/resolv.conf.d/head* file:

```
# DO NOT CHANGE THIS FILE DIRECTLY. YOUR MODIFICATIONS WILL BE OVERWRITTEN.
#
# For static IP configurations, nameservers can be specified via `ghe-setup-network'.
# For DHCP configurations, nameservers are auto-populated with values provided in the DHCP lease.
# To add custom options to this file, edit /etc/resolvconf/resolv.conf.d/head and run `sudo service resolvconf restart'.
#
# The 127.0.0.1 entry is *required* to be the first entry in the list here. Removing it
# or having another entry as the first entry will break GitHub Enterprise. It's also not
# supported to add options that affect the ordering here such as "options rotate".
```

But the same content should be documented. Ideally we should have something like this:

image

### PR DESCRIPTION
**Pull request by riteshsp2000**
### Why:
On configure DNS Nameservers page, add note preventing user from changing nameservers.
Closes #11080 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@riteshsp2000 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: data/reusables/enterprise_installation/preventing-nameservers-change.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+	{% note %}
+
+	**Note**: The 127.0.0.1 entry is **required** to be the first entry in the list here. Removing it or having another entry as the first entry will break GitHub Enterprise. It's also not supported to add options that affect the ordering here such as "options rotate"
```
Comment (lecoursen) [inline data/reusables/enterprise_installation/preventing-nameservers-change.md:3]:
```suggestion
	**Note**: The 127.0.0.1 entry is required to be the first entry in the list. Do not remove the 127.0.0.1 entry, add another entry above the 127.0.0.1 entry, or use options that affect the ordering of entries, such as "options rotate".
```

**Review by lecoursen — DISMISSED** [review]
Thanks @riteshsp2000! I'm going to apply a small change for style, then I'll merge. ⚡

**Comment by riteshsp2000** [comment]
> Thanks @riteshsp2000! I'm going to apply a small change for style, then I'll merge. ⚡ 

@lecoursen anything that I can help you with?

**Comment by lecoursen** [comment]
> @lecoursen anything that I can help you with?

Nope, but I misspoke when I said I'd merge now. Right now the repository is frozen, so we'll have to wait to merge until after it's thawed. Someone on our team will take care of that though!

**Review by ramyaparimi — APPROVED** [review]

**Pull request merged**